communitiesuk/funding-service-design-assessment-store
# Fixing script for calculating total

FS-3073 https://dluhcdigital.atlassian.net/browse/FS-3073

### Change description
Changing the Import applications to assessment methods to correctly calculate the funding total



### How to test
Have a complete application and run the script, check the funding total is correct for the amounts you inputted on the funding required form


### Screenshots of UI changes (if applicable)
![image](https://github.com/communitiesuk/funding-service-design-assessment-store/assets/97108643/99f9b3cf-cfc4-418f-96c7-e9ddc01fb317)
